UM/Qt/Bindings/ProfilesModel.py

```diff
@@ -54,9 +61,22 @@ def setAddUseGlobal(self, add):
     def addUseGlobal(self):
         return self._add_use_global
 
+    addWorkingProfileChanged = pyqtSignal()
+
+    def setAddWorkingProfile(self, add):
+        if add != self._add_working_profile:
+            self._add_working_profile = add
+            self._onProfilesChanged()
+            self.addWorkingProfileChanged.emit()
```

Comment:
I'm not sure what this function and the above pyqtSignal does. Perhaps you could document at least what the signal is? Or what an "add working profile" is as a noun?

But it seems that `addWorkingProfileChanged` and `setAddWorkingProfile` have some sub-par names in any case.